homeassistant/components/binary_sensor/ring.py

```diff
@@ -73,6 +73,7 @@ def __init__(self, hass, data, sensor_type):
                                       SENSOR_TYPES.get(self._sensor_type)[0])
         self._device_class = SENSOR_TYPES.get(self._sensor_type)[2]
         self._state = None
+        self._unique_id = '{}-{}'.format(self._data.id, self._sensor_type)
```

Comment:
Might make more sense to have the sensor type before the id when computing the unique id?

Reply:
@rohankapoorcom I was going by the dev docs here where they recommend to place the sensor after the ID.  The ID itself is provided by the 

https://developers.home-assistant.io/docs/en/entity_registry_index.html#unique-id-requirements

Reply:
Aha, I read the docs wrong :(